docs/content/direct/user-guide.md

```diff
@@ -12,6 +12,62 @@ See [Using an existing hosting cluster](./hosting-cluster.md)
 
 Thus far we can only say how to handle this when the hosting cluster is OpenShift. The problem is getting URLs that work from everywhere. OpenShift is a hosted product, your clusters have domain names that are resolvable from everywhere. In other words, if you use an OpenShift cluster as your hosting cluster then this problem is already solved.
 
+## WEC-independent workload object transformation
+
+KubeStellar does some transformation of workload objects on their way from WDS to WEC. First, there are transformations that are independent of the destination; these are described in this section. Second, there is customization to the WEC, described [later](#rule-based-customization).
+
+The WEC-independent transformations are removal of certain content.
+
+There are three categories of these transformations, as follows. They are applied in this order.
+
+1. Transformations that are built into KubeStellar and apply to all workload objects.
+1. Transformations that are built into KubeStellar and apply to specific kinds of workload objects.
+1. Transformations that are configured by control objects and apply to specific kinds of workload objects.
+
+### Transformations for all workload objects
+
+The following are applied to every workload object.
+
+1. Remove the following fields from `metadata`: `managedFields`, `finalizers`, `generation`, `ownerReferences`, `selfLink`, `resourceVersion`, `UID`, `generateName`.
+1. Remove the annotation named `kubectl.kubernetes.io/last-applied-configuration`.
+1. Remove the `status`.
+
+## Built-in transformations of specific kinds of workload object
+
+In a `Service` (core API group) object, remove the following fields from `spec`: `clusterIP`, `clusterIPs`, `ipFamilies`, `externalTrafficPolicy`, `internalTrafficPolicy`, `ipFamilyPolicy`, `sessionAffinity`. Also remove the `nodePort` field from every port unless the annotation `kubestellar.io/annotations/preserve=nodeport` is present.
```

Comment:
#2070 changes this. The second PR among these two to merged should first be updated accordingly.